### PR TITLE
Fix #21837: Formatter being overwritten - RideConstruction.cpp

### LINKED ISSUE #21837: Formatter seems overwritten before being used
https://github.com/OpenRCT2/OpenRCT2/blob/9b7face116a7d8b83f5c092511be74092af3b95c/src/openrct2-ui/windows/RideConstruction.cpp#L1505-L1541

In the code above in line 1505 formatter `ft` is created and then values are added to it in 1506, 1516, but then in 1539 `ft` is assigned to again and its values consumed in 1541.

It seems the code in 1505, 1506 and 1516 is unnecessary.

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
Fix https://github.com/OpenRCT2/OpenRCT2/issues/21837 - remove unnecessary lines as the value in the Formatter is overwritten.

### CODE REVIEW

**Review by janisozaur — APPROVED** [review]

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
This is probably not going to work and break the display of that window. It uses the common formatter, which means that overwriting it will not erase the data, it will only reset the internal counter.

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 1]
```diff
@@ -1536,7 +1536,6 @@ static Widget _rideConstructionWidgets[] = {
             }
 
             // Set window title arguments
-            ft = Formatter::Common();
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/RideConstruction.cpp:1539]: Again, have you tested this?

**Comment by Harry-Hopkinson** [comment]
After further investigation - it removes the name of the ride at the top of the RideConstructionWindow.

**Comment by Harry-Hopkinson** [comment]
@Gymnasiast After testing - using ```ft.Rewind()``` does work. Thanks and sorry that I did not pick up the change removing the ride title when I was testing.

**Review by Gymnasiast — APPROVED** [review]

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
I’d like to check a thing - I think there is a more elegant solution here.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**